### ISSUE TITLE
Doubling all goldens in my collection + extra Priest Goldens

### ISSUE THREAD

**Issue by AshtonSanders** [ko-vasilev/Hearthstone-Collection-Tracker#86]
It looks like the new dark background color for "all cards" is doubling all goldens in my collection. 

The most recent scan also gave me two golden shifting shades, power word tentacles, hooded acolyte, and twilight darkmender (which I don't have any goldens of).

**Comment by ko-vasilev** [comment]
Yeah, I thought there might be new bugs due to colors changed in the collection.

**Comment by scottismyname** [comment]
Also, I think any card that it detects one of, it detects 2 instead.  So when it's finished running, it either detects 0 or 2 of each card, never 1.

**Comment by tducote** [comment]
This bug still persists. However, if the import is done on a single set at a time, as long as the set isn't a wild set (GvG), the import seems to work just fine since the background color is not the darker color.
